tests/models/opt/test_embeddings_opt.py

```diff
@@ -0,0 +1,66 @@
+# coding=utf-8
+# Copyright 2021, The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" Testing suite for the PyTorch OPT model. """
+
+import unittest
+
+
+from transformers import OPTConfig, is_torch_available, BartTokenizerFast
+
+
+
+if is_torch_available():
+    import torch
+
+    from transformers import (
+        AutoModelForSequenceClassification,
+        OPTForCausalLM,
+        OPTForConditionalGeneration,
+        OPTForQuestionAnswering,
+        OPTForSequenceClassification,
+        OPTModel,
+        OPTTokenizer,
+        pipeline,
+    )
+
+class OPTEmbeddingsTest(unittest.TestCase):
```

Comment:
Can we add this as an integration test under `test_modeling_opt.py`?

Reply:
+ we should also add a generation test

Reply:
resolved here 842727999cb6d7a298963021dcd5daafb98d2453